chemprop/train/run_training.py

```diff
@@ -200,7 +200,12 @@ def run_training(args: TrainArgs,
         else:
             debug(f'Building model {model_idx}')
             model = MoleculeModel(args)
-
+            
+        # Optionally, overwrite weights:
+        if args.frzn_mpnn | (args.frzn_ffn_layers > 0):
+            debug(f'Loading mpn parameters from {args.checkpoint_frzn}')
```

Comment:
I might specify in this log message that it is the frozen parameters. And maybe since it is mpn for ffn parameters, just saying model parameters generally.